### PR TITLE
Detailed list of third-party libraries

### PR DESCRIPTION
**Pull request by kenan435**
Detailed list of all the third-party libraries being used in this project.

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# Credits
+
```
Comment (matrixik) [inline NOTICE.md]: Add short summary what this list is (even copy-paste of commit subject is fine).

**Comment by kenan435** [comment]
Done...

File: NOTICE.md (hunk 1 above)
Comment (kenan435) [reply NOTICE.md]: Done.

File: LICENSES-NOTICE.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
+# Credits
+Detailed list of all the third-party libraries being used in this project
```
Comment (matrixik) [inline LICENSES-NOTICE.md]: I would complain about you did not adding dot at the end but you probably already hate me so merging as it is.

**Comment by matrixik** [comment]
Thank you.

File: LICENSES-NOTICE.md (hunk 2 above)
Comment (kenan435) [reply LICENSES-NOTICE.md]: :) Yesterday I had to take out a dot from a commit message subject. Thx for merging it.

**Pull request merged**